src/main/java/nl/knaw/dans/easy/sword2examples/Common.java

```diff
@@ -59,44 +63,49 @@ public static String readEntityAsString(HttpEntity entity) throws IOException {
         return new String(bos.toByteArray(), "UTF-8");
     }
 
-    /**
-     * Returns the result of evaluating the XPath expression <code>expr</code> on the XML code in the string <code>xml</code>
-     *
-     * @param xml
-     *        the XML to query
-     * @param expr
-     *        the XPath expression
-     * @return the result as a string
-     * @throws Exception
-     */
-    public static String getStringFromXml(String xml, String expr) throws Exception {
-        InputSource xmlSource = new InputSource(new StringReader(xml));
-        XPathFactory xpathFactory = XPathFactory.newInstance();
-        XPath xpath = xpathFactory.newXPath();
-        return (String) xpath.evaluate(expr, xmlSource, XPathConstants.STRING);
+    public static Entry parseEntry(String text) {
+        Abdera abdera = Abdera.getInstance();
+        Parser parser = abdera.getParser();
+        Document<Entry> receipt = parser.parse(new StringReader(text));
+        return receipt.getRoot();
     }
 
-    static void trackDeposit(CloseableHttpClient http, URI statIri) throws Exception {
+    public static Feed parseFeed(String text) {
+        Abdera abdera = Abdera.getInstance();
+        Parser parser = abdera.getParser();
+        Document<Feed> receipt = parser.parse(new StringReader(text));
+        return receipt.getRoot();
+    }
+
+    static void trackDeposit(CloseableHttpClient http, IRI statIri) throws Exception {
         CloseableHttpResponse response;
         String bodyText;
         System.out.println("Start polling Stat-IRI for the current status of the deposit, waiting 10 seconds before every request ...");
-        String state = null;
+        String stateTerm = null;
```

Comment:
(10 minutes later...) Actually, now that I look at it again, you don't need these declarations here at all!